src/Features/CSharp/Portable/PdbSourceDocument/CSharpPdbSourceDocumentNavigationService.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.PdbSourceDocument;
+
+namespace Microsoft.CodeAnalysis.CSharp.PdbSourceDocument
+{
+    [ExportLanguageService(typeof(IPdbSourceDocumentNavigationService), LanguageNames.CSharp), Shared]
+    internal class CSharpPdbSourceDocumentNavigationService : AbstractPdbSourceDocumentNavigationService
```

Comment:
>CSharpPdbSourceDocumentNavigationService [](http://example.com/codeflow?start=19&length=40)

This is odd. The language of the source we are navigating to is not determined by the language we are navigating from. I don't think it makes sense for this service to be language specific.